docs/kubernetes/KubernetesPolicy.md

```diff
@@ -0,0 +1,62 @@
+# Programming Calico Policy in Kubernetes
+The [Calico-Kubernetes plugin](https://github.com/projectcalico/calico-docker/blob/master/docs/kubernetes/KubernetesIntegration.md) will allow you to program Networking policy directly from your Kubernetes pod config. 
+
+
+### The Basics
+The Calico Networking plugin will look for a `policy` key in the annotations section of your pod's metadata. This `policy` key should map to a single string that outlines networking behavior semantically.
+
+Programming policy is as simple as saying what you want your pod to do!
+
+For instance:
+```
+...
+  metadata:
+    annotations:
+      policy: "allow tcp from label name=backend"
+...
+```
+
+To specify multiple rules, separate them by semicolons.
+```
+...
+      policy: "allow from label name=backend; allow tcp to ports 4001,443"
+...
+```
+In it's current state, Calico supports whitelist oriented, inbound rules. This means that anything unspecified by the policy spec will be rejected at the destination. 
+
+Calico can specify any combination of label (multiple labels per rule are unsupported), protocol, source/dest ports, and source/dest net. The full syntax for creating compound rules is shown below [below](#Syntax).
+
+### Defaults
+With no specified policy, Calico will only allow traffic from within a pod's own namespace. This default rule will be overidden if any policy is programmed. If you wish to include rules that police the namespace, you can do so with the keyword `tag namespace_<NAMESPACE>`, as in `allow tcp from tag namespace_default`
```

Comment:
Eh, I don't think we want to be exposing the "allow from namespace" semantics to the world.  That is more of a fluke in terms of how we implemented the first version, rather than an approved and supported way of declaring policy.

We still need to design and propose the correct way of exposing services across namespaces.  For now, it is unsupported.